package.json

```diff
@@ -50,7 +50,7 @@
     "apis-helpers": "0.0.1"
   },
   "scripts": {
-    "test": "node_modules/mocha/bin/mocha test/integration",
+    "test": "mocha test/integration",
```

Comment:
This means that you have to install `mocha` globally. Can we add that somehow in the `package.json` or do we just need to include that in the `README` setup instructions?